analyzers/nodejs/nodejs_test.go

```diff
@@ -202,10 +227,35 @@ func TestUsingYarnLockfileFallback(t *testing.T) {
 	assert.NoError(t, err)
 
 	chaiProject := analysisResults.Transitive[chaiDirectDep.Resolved]
-	AssertImport(t, chaiProject.Imports, "assertion-error", "1.1.0")
-	AssertImport(t, chaiProject.Imports, "check-error", "1.0.2")
-	AssertImport(t, chaiProject.Imports, "get-func-name", "2.0.0")
-	AssertImport(t, chaiProject.Imports, "pathval", "1.1.0")
-	AssertImport(t, chaiProject.Imports, "deep-eql", "3.0.1")
-	AssertImport(t, chaiProject.Imports, "type-detect", "4.0.8")
+	assertImport(t, chaiProject.Imports, "assertion-error", "1.1.0")
+	assertImport(t, chaiProject.Imports, "check-error", "1.0.2")
+	assertImport(t, chaiProject.Imports, "get-func-name", "2.0.0")
+	assertImport(t, chaiProject.Imports, "pathval", "1.1.0")
+	assertImport(t, chaiProject.Imports, "deep-eql", "3.0.1")
+	assertImport(t, chaiProject.Imports, "type-detect", "4.0.8")
+}
+
+func assertPackageReturnImports(t *testing.T, packages map[pkg.ID]pkg.Package, name, revision string) pkg.Package {
```

Comment:
If you're still using this, I would update it to not execute an assertion. Keeping SRP (single responsibility principle) in mind, just fetch the package. Run the assertion on the correctness of the result outside of this scope